spec/features/user_registers_new_application_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.feature "User registers new application", js: true do
+  scenario "successfully" do
+    user = create(:user)
+
+    visit new_user_session_path
+    fill_in "Email", with: user.email
+    fill_in "Password", with: user.password
+
+    click_button "Log in"
+
+    expect(page).to have_title("Registered Applications")
+    expect(page).to have_content("New Application")
```

Comment:
The expectation on line 14 is also covered by the click_link on line 16 as both rely on the same text being visible on the page.